### ISSUE TITLE
Documentation suggests using deprecated next-transpile-modules

### ISSUE THREAD

**Issue by wojtekmaj** [PiwikPRO/next-piwik-pro#12]
next-transpile-modules package has been deprecated with the following message:

> All features of next-transpile-modules are now natively built-in Next.js 13.1. Please use Next's transpilePackages option :)

**Comment by ciruz** [comment]
@wojtekmaj do you use this package with next 13 + app directory?

**Comment by wojtekmaj** [comment]
No, with the old way. Is that relevant?

**Comment by ciruz** [comment]
No, I thought of something else. I'm going to implement Piwik into next.js with the `/app` directory, and I assumed maybe you have already done this. NVM. It's not relevant to this issue. Not going to hijack this thread any further.

**Comment by mwormgoor** [comment]
The solution to use the built-in version is documented here:
https://github.com/martpie/next-transpile-modules/releases/tag/the-end

1. uninstall next-transpile-modules
2. update next.config.js as shown in the URL pasted above

**Comment by lysy-vlc** [comment]
Already resolved. Please check the newest version
